rakefile

```diff
@@ -64,6 +67,16 @@ rototiller_task :performance do |t|
         end
       end
     end
+    unless ENV['BEAKER_TESTS'] == ''
```

Comment:
does this not need a check for nil?

Reply:
This is basically just to make this optional, if it is nil, then it will get the default value.  If you don't want to run any tests, you have to explicitly set the BEAKER_TESTS env var to an empty string.

Reply:
true

Reply:
This is good.  I was sad when I first found out that beaker didn't have a -notests arg...  and that by default even setting it to '' doesn't stop it from running the default tests.

Reply:
you can just leave --tests off.  often i `touch empty.rb` and use that as my "test", ie when a rake task implies --test

Reply:
yeah, I have it a bit confused in my head.  It was that without the if he has here, you can't turnt he tests off in a simple intuitive way, like setting the var to ''.  Seems to me if I pass --tests '' to beaker, it shouldn't run tests.